### ISSUE TITLE
Detailed documentation is missing from ReadTheDocs

### ISSUE THREAD

**Issue by kmatch98** [adafruit/Adafruit_CircuitPython_DisplayIO_Layout#34]
I noticed that the detailed documentation from the docstrings is no longer being generated into the ReadTheDocs. 

It's missing the graphics and all the long-form description of how these widgets are made.  

- One example is the `Widget` class definition that had a lot of details about how to use it.  
- Another example is the `Dial` widget where the graphics are missing.

**Q. Is it worth resolving this now or wait until these find their new home in the CircuitPython github pages?**

**Comment by jposada202020** [comment]
Interesting I have the version of my repo for in readthedocs and is showing fine. 
https://circuitpython-displayio-jdm.readthedocs.io/en/latest/api.html

Maybe changes introduces after https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_Layout/commit/f5ab9c85549875db2cee020d1b6f730f252478af

I verify one of the libraries that I have updated last week an is showing fine with the nice text and verything.

**Comment by jposada202020** [comment]
Could this be related
![image](https://user-images.githubusercontent.com/34255413/115720183-ce5c6080-a34a-11eb-9306-75bfb19914a3.png)

**Comment by kmatch98** [comment]
I tried building the docs locally and they failed due to the missing "autodoc_mock_imports" in the `conf.py` file with error message:
```
Warning, treated as error:
autodoc: failed to import module 'grid_layout' from module 'adafruit_displayio_layout.layouts'; the following exception was raised:
No module named 'displayio'
```

Could this have been caused by: https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_Layout/commit/cd8728afae72d5bfa9e648ffb86ef9565f28ea25

Is there something that needs to be configured on my local machine to work like the docs building on the CI is working?

**Comment by jposada202020** [comment]
As I suffered a lot with this, only the library requirements, But I have seen differences in the builts in the way the CI  and the locally. As proven in color picker library docs. 
I could bump my repo, and rebuild to see if the changes in https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_Layout/commit/cd8728afae72d5bfa9e648ffb86ef9565f28ea25 are the cause, what do you think @kmatch98 ? (just do not want to loose the beautiful documentation 😄 )

**Comment by jposada202020** [comment]
@kmatch98 I Merged the changes and the nice documentation is still there and it works, so the changes are not the problem. Maybe something in the CI
https://circuitpython-displayio-jdm.readthedocs.io/en/latest/api.html#dial

**Comment by FoamyGuy** [comment]
If we can find an easy fix for this repo we could do it. But I think it's okay not to dig too deep here and focus on the new repos instead. The good news is the first few of those do seem to render the docs correctly with the images for me locally, so perhaps whatever is the cause is already fixed there.

**Comment by jposada202020** [comment]
This is solved by #36. Clsoing
